tools-and-tests/k6/package.json

```
@@ -0,0 +1,10 @@
+{
+  "name": "k6",
```

We must apply convention here even if we don't publish.
Also see https://github.com/hiero-ledger/hiero-mirror-node/blob/main/rest/package.json for a template of what this file should look like.
```suggestion
  "name": "@hiero-ledger/block-node-k6",
```

---

If we are to keep these dev dependencies, we can create these conventions, otherwise if we delete these, the change is not needed. I see for and against opinions on deleting the package.json.

---

We have to do it as part of Hiero compliance and best practices. Similar to having the license information on files.
You should only need to see a few fields to be proper

---

Noted about the Hiero compliance.

---

On the note of having the license headers in the files, should we add these as well to the new k6 tests? Also, is it possible to enable spotless to also run on the js/ts files? That would be great!